services/web/client/source/class/osparc/desktop/WorkbenchView.js

```diff
@@ -667,6 +667,26 @@ qx.Class.define("osparc.desktop.WorkbenchView", {
           }
         }, this);
       }
+
+      // callback for events
+      const slotName4 = "event";
+      if (!socket.slotExists(slotName4)) {
+        socket.on(slotName4, eventData => {
+          const eventPayload = JSON.parse(eventData);
+          const action = eventPayload["action"];
+          if (action == "RELOAD_IFRAME") {
+            // TODO: maybe reload iframe in the future
+            // for now a message is displayed to the user
+            const nodeId = eventPayload["node_id"];
+
+            const workbench = this.getStudy().getWorkbench();
+            const node = workbench.getNode(nodeId);
+            const label = node.getLabel();
+            const text = `New inputs for service ${label}. Please reload to refresh service.`;
```

Comment:
what does that mean? do I need to manually refresh the UI or will it be automatic?

Reply:
After talking with @odeimaiz  he suggested to have the user manually refresh the UI.